src/lib/_imports/components/c-card.css

```diff
@@ -104,8 +104,10 @@
 
 /* Elements */
 
-:--c-card > a:not(:--c-button), :--c-card > p > a:only-child:not(:--c-button) {
-    display: flex;
+:--c-card > a:not(:--c-button),
+:--c-card > p > a:not(:--c-button) {
+    display: inline-flex;
+    align-items: baseline;
 }
```

Comment:
@R-Tomas-Gonzalez, regarding [adding space between certain links](https://github.com/TACC/Core-Styles/pull/293#pullrequestreview-1811686497), I came up with this:

| wide card | narrow card |
| - | - |
| <img width="960" alt="space between p   a links, wide card" src="https://github.com/TACC/Core-Styles/assets/62723358/e3fbcb11-f421-4477-bc7f-78ceced5785e"> | <img width="960" alt="space between p   a links, narrow card" src="https://github.com/TACC/Core-Styles/assets/62723358/bc87990f-386c-4ad2-a0a5-816bae76846b"> |

<details><summary>thoughts and failures leading to that solution</summary>

1. I would like to.
2. Using `margin-left` on link causes a bug.
3. Using `margin-right` on link causes a bug.
4. Using `display: flex` on parent requires an improbable selector.
5. Only use case I found is solved with a " | " text between the links.
6. But what if I assumed all such links have an icon...

| Thought | Example |
| - | - |
| 2 | <img width="960" alt="use margin-left, causes indent" src="https://github.com/TACC/Core-Styles/assets/62723358/2144f592-e903-4699-bc9c-30451f23ef8f"> |
| 3 | <img width="960" alt="use margin-right, exists within paragraph maybe" src="https://github.com/TACC/Core-Styles/assets/62723358/b16effb2-0b73-42d2-b66e-963323965129"> |
| 4 | <img width="960" alt="use flex, but how to select parent" src="https://github.com/TACC/Core-Styles/assets/62723358/79e792ba-77b8-422a-babb-227d3a429d30"> |
| 5 | <img width="960" alt="only similar use case I found is solved" src="https://github.com/TACC/Core-Styles/assets/62723358/d59be3bc-47c1-47a0-a690-20c8a61c54a9"> |

</details>

Reply:
Looks good!